chef/cookbooks/dhcp/libraries/helpers.rb

```diff
@@ -0,0 +1,9 @@
+module DhcpHelper
+  def self.config_filename(base, ip_version, extension = ".conf")
+    if ip_version == "4"
+      "#{base}#{extension}"
+    else
+      "#{base}#{ip_version}#{extension}"
+    end
```

Comment:
To be fair I had envisioned this as always returning "#{base}#{extra}#{extension}" where extra gets set to 6 if ip_version is 6, but this is very much a nit.